pkg/util/pretty/spinner.go

```diff
@@ -9,9 +9,10 @@ import (
 // Spinner style.
 //
 // Usage:
-//   sp, _ := pretty.Spinner.Start("Starting ...")
-//   time.Sleep(time.Second * 3)
-//   sp.Success("Done")
+//
```

Comment:
double check: https://pkg.go.dev/kusionstack.io/kusion@v0.7.2/pkg/util/pretty

![image](https://user-images.githubusercontent.com/23924124/206347570-0d76cb7c-9c4f-4a40-a122-7f7a466433e8.png)

Reply:
Same format issue. It is recognized as blocker format issue in golangci-lint v1.50.1.